include/deal.II/base/parameter_handler.h

```diff
@@ -67,10 +67,19 @@ namespace Patterns
      */
     virtual bool match (const std::string &test_string) const = 0;
 
+    enum OutputStyle
+    {
+      // Machine interpretable format, default and used for create
+      Machine,
```

Comment:
Ah, I see. In that case leave it as is. You may want to reference `ParameterHandler::OutputStyle` in the documentation of the current enum.